plugin/src/main/java/com/iridium/iridiumcore/utils/SkinUtils.java

```diff
@@ -52,6 +56,7 @@ public static UUID getUUID(String username) {
 
     public static String getHeadData(UUID uuid) {
         if (uuid.equals(loadingUUID)) return steveSkin;
+        if (uuid.toString().contains("00000000-0000-0000")) return steveSkin; // Bedrock Player (GeyserMC)
```

Comment:
Make this `"00000000-0000-0000"` a field, maybe? With that, you might be able to drop the comment too.